### PR TITLE
lost connection reconnection test

### PR DESCRIPTION
**Pull request by mustafaiman**

### CODE REVIEW

**Comment by GhostInAMachine** [comment]
Right now when client authenticates to a member other than the first one it does not set owner connection flag to false, meaning that all connections are owner connections. I have a WIP branch somewhere, let me fix it and I will merge this PR later.

**Comment by mustafaiman** [comment]
I also saw that issue. Glad to hear you have a fix for that. Looking forward to merge that.

**Comment by mustafaiman** [comment]
since we merged https://github.com/hazelcast/hazelcast-nodejs-client/pull/76. How about now? @GhostInAMachine

**Comment by GhostInAMachine** [comment]
Does the test still pass?

**Comment by mustafaiman** [comment]
yes

File: test/LostConnectionTest.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    });
+
+    it('M2 starts, M1 goes down, client sets M2 as owner', function(done) {
+        this.timeout(12000);
+        var newMember;
+        client.clusterService.on('memberAdded', function() {
+            Controller.shutdownMember(cluster.id, member1.uuid).then(function() {
+                setTimeout(function() {
+                    try {
+                        expect(client.clusterService.getOwnerConnection().address.port).to.be.eq(newMember.port);
```
Comment (GhostInAMachine) [inline test/LostConnectionTest.js]: where is the host comparison?

**Comment by mustafaiman** [comment]
added host comparison

**Pull request merged**